packages/gensx/tests/checkpoint.test.tsx

```diff
@@ -357,4 +362,107 @@ suite("checkpoint", () => {
       ],
     });
   });
+
+  test("handles streaming components", async () => {
```

Comment:
I think this should be two tests.

Reply:
Good point, fixed via https://github.com/cortexclick/gensx/pull/155/commits/994e0a8016ad46d0ad0e138fc95c0a5dc0dc186d